src/main/resources/application.yml

```diff
@@ -47,12 +52,6 @@ spring:
       connection-timeout: 1000
       validation-timeout: 500
 
-  flyway:
```

Comment:
So `flyway` is not used anymore or it's clever enough to get the settings from the `datasource` properties?